src/main/resources/com/mathworks/ci/BuildArtifactAction/index.jelly

```diff
@@ -0,0 +1,92 @@
+
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define"
+  xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:r="/lib/hudson/test" xmlns:p="/lib/test">
+
+    <l:layout title="MATLAB Build Results">
+        <st:include  it="${it.owner}" page="sidepanel.jelly"/>
+
+    <l:main-panel>
+        <j:set var="st" value="${it.status}" />
+        <h1 class="${st.cssClass}">
+            <st:out value="MATLAB Build Results" />
+        </h1>
+        <div>
+            <j:choose>
+                <j:when test="${it.totalCount==0}">
+                    No tasks
+                </j:when>
+                <j:otherwise>
+                    <div>
+                        ${(it.totalCount)} Tasks
+                        <j:if test="${prev!=null}">
+                            (${h.getDiffString(it.failCount-prev.failCount)})
+                        </j:if>
+                        <!-- Only show skips if there are some since they don't make sense for JUnit, only for TestNG. -->
+                        <j:if test="${it.skipCount > 0}">
+                            , Skipped ${(it.skipCount)}
+                            <j:if test="${prev!=null}">
+                                (${h.getDiffString(it.skipCount-prev.skipCount)})
+                            </j:if>
+                        </j:if>
+                    </div>
+                    <div style="width:100%; height:1em; background-color: #729FCF; border-radius: 6px; overflow: hidden;">
+                        <!-- Failed task part of the bar. -->
+                        <div style="width:${it.failCount*100/it.totalCount}%; height: 1em; background-color: #EF2929; float: right;"></div>
+                        <!-- Skipped task part of the bar. -->
+                        <div style="width:${it.skipCount*100/it.totalCount}%; height: 1em; background-color: #FCE94F; float: right;"></div>
+                    </div>
+                    <div align="right">
+                        ${(it.failCount)} Failures
+                        <j:if test="${prev!=null}">
+                            (${h.getDiffString(it.totalCount-prev.totalCount)})
+                        </j:if>
+                    </div>
+                </j:otherwise>
+            </j:choose>
+        </div>
+
+        <table class="jenkins-table sortable" id="buildresults">
+            <thead>
+                <tr>
+                    <th class="pane-header" style="width:6em">Task Name</th>
+                    <th class="pane-header" style="width:6em">Status </th>
+                    <th class="pane-header">Description</th>
+                    <th class="pane-header" style="width:6em">Duration (HH:MM:SS)</th>
```

Comment:
If you use the "HH:MM:SS" format to display run times, then remove the (ms) unit from the corresponding column header.

Reply:
You mean should i remove (ms) string from jelly ? line 55 is the column header.

Reply:
Yes, in this screenshot (ms) looks redundant:

![image](https://github.com/mathworks/jenkins-matlab-plugin/assets/48831250/0801c002-c0b8-4112-bf41-6c9b938c8e5b)

Reply:
Ohh i guess the Screeshot was old. I have already removed it from the Jelly file